deployments/liqo/templates/liqo-crd-replicator-deployment.yaml

```diff
@@ -50,7 +50,7 @@ spec:
           resources: {{- toYaml .Values.crdReplicator.pod.resources | nindent 12 }}
           ports:
           - name: metrics
-            containerPort: 8082
```

Comment:
Hi @kariya-mitsuru, thanks for your PR. Can you keep the "8082" port and add a flag inside the CRD replicator to set the port, please?

Could you edit /cmd/crd-replicator/main.go.

Check this https://github.com/liqotech/liqo/blob/c2f889a26d38a1785310b56290c28419b8892761/cmd/liqo-controller-manager/main.go#L106 to have a guideline about how to implement it.